plugin/trino-hive/src/test/java/io/trino/plugin/hive/fs/BaseCachingDirectoryListerTest.java

```diff
@@ -315,13 +315,13 @@ public void testDropPartitionedTable()
         String tableGroup1PartitionLocation = getPartitionLocation(TPCH_SCHEMA, "drop_partitioned_table", ImmutableList.of("group1"));
         String tableGroup2PartitionLocation = getPartitionLocation(TPCH_SCHEMA, "drop_partitioned_table", ImmutableList.of("group2"));
         String tableGroup3PartitionLocation = getPartitionLocation(TPCH_SCHEMA, "drop_partitioned_table", ImmutableList.of("group3"));
-        assertThat(isCached(tableGroup1PartitionLocation)).isTrue();
-        assertThat(isCached(tableGroup2PartitionLocation)).isTrue();
-        assertThat(isCached(tableGroup3PartitionLocation)).isTrue();
+        assertThat(isCached(tableGroup1PartitionLocation, SchemaTableName.schemaTableName(TPCH_SCHEMA, "drop_partitioned_table"))).isTrue();
+        assertThat(isCached(tableGroup2PartitionLocation, SchemaTableName.schemaTableName(TPCH_SCHEMA, "drop_partitioned_table"))).isTrue();
+        assertThat(isCached(tableGroup3PartitionLocation, SchemaTableName.schemaTableName(TPCH_SCHEMA, "drop_partitioned_table"))).isTrue();
         assertUpdate("DROP TABLE drop_partitioned_table");
-        assertThat(isCached(tableGroup1PartitionLocation)).isFalse();
-        assertThat(isCached(tableGroup2PartitionLocation)).isFalse();
-        assertThat(isCached(tableGroup3PartitionLocation)).isFalse();
+        assertThat(isCached(tableGroup1PartitionLocation, SchemaTableName.schemaTableName(TPCH_SCHEMA, "drop_partitioned_table"))).isFalse();
+        assertThat(isCached(tableGroup2PartitionLocation, SchemaTableName.schemaTableName(TPCH_SCHEMA, "drop_partitioned_table"))).isFalse();
+        assertThat(isCached(tableGroup3PartitionLocation, SchemaTableName.schemaTableName(TPCH_SCHEMA, "drop_partitioned_table"))).isFalse();
```

Comment:
Can we add an additional assertion which ensure other tables are not removed ? Or was it added ?

Reply:
good call, ill add verification to this test.